<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1367</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gismo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser_output">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>1171</width>
      <height>661</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ISOCTEUR</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'ISOCTEUR'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.25pt;&quot;&gt;a---as&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.25pt;&quot;&gt;0-----&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_serch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ISOCPEUR</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>12x25 rectangle - 209j
12x25 rectangle - 101i
12x25 rectangle - 1109j
10 dia bar - 965j
10 dia bar - 972j
10 dia bar - 757i</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Report">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string> Report</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Fx_My">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string> Fx-My</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Input list:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Fx_Mz">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string> Fx-Mz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Fx_Mtot">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Fx-Mtot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Fy_Fz">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string> |Fz|-|Fy|</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Mx_Vtot">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>|Mx|-Vtot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_info">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>App info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_print">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Print report</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Check list</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clbResults">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt; get beam results</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Sort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort list</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_idea">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>include Idea with factor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ideafactor">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Clipboard data:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clbMembers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt; get beam numbers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clbNodes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt; get node numbers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_getMembers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>get all members</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_makei">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string> make all  i</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_makej">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>make all j</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_makeij">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>make all i and j</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_My_Mz">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mz-My</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_full">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>full report</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_pltAnnot">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>10</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>plot annotation</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_staadname">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>710</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>710</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data source staad file name:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_normFy_Fz">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>norm Fz-Fy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>550</y>
      <width>100</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Set preset:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>570</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>60</y>
      <width>102</width>
      <height>483</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Content:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_idea_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>maxFx</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_idea_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>minFx</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_idea_4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>max|Fy|</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_idea_6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>maxFynorm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_idea_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>max|Fz|</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_idea_7">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>maxFznorm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_idea_16">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>max |Vtot|</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_idea_8">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>max|Mx|</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_idea_9">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>max|My|</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_idea_10">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>maxMy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_idea_11">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>minMy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_idea_14">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>max|Mz|</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_idea_12">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>maxMz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_idea_13">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>minMz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_idea_15">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>max |Mtot|</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_idea_17">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>max conn comp </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_idea_18">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>max bolt tens</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_idea_19">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>max bolt shear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1367</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <action name="actionLoadXLS">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Load xls data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
